crypto/CMakeLists.txt

```diff
@@ -193,7 +193,7 @@ if(HOST_ASM_MINGW64_X86_64)
 		whrlpool/wp-mingw64-x86_64.S
 		cpuid-mingw64-x86_64.S
 	)
-	add_definitions(-Dendbr64=)
```

Comment:
Don't forget crypto/Makefile.am.mingw64-x86_64 too

Reply:
Thanks for the reminder, I applied the same for autotools.